### ISSUE TITLE
I've found that the snpe backend has been supported, how about tensorflow lite discussion result?

### ISSUE THREAD

**Issue by starlitsky2010** [open-mmlab/mmdeploy#967]
**Describe the feature**
Tensorflow lite is widely used by edge side.
Taking advantage of post-training integer quantization it can run quantization model on CPU runtime. But SNPE can't do this.

**Motivation**

To make full use of low level hardware on different chipsets.

**Related resources**

https://tensorflow.google.cn/lite/performance/model_optimization

**Additional context**

So I wonder if there is a plan for tensorflow lite support?

Thanks

**Comment by lvhan028** [comment]
Are you suggesting training model by PyTorch and deploying model by TensorFlow? 
In this case, I am afraid TFLite is not in our plan.
BTW, could you tell us your interested devices?
